.travis.yml

```diff
@@ -49,8 +49,8 @@ before_script:
     fi
   - |
     if [[ "$WP_TRAVISCI" == "phpcs" ]] ; then
-      composer global require wp-coding-standards/wpcs
-      phpcs --config-set installed_paths $HOME/.composer/vendor/wp-coding-standards/wpcs
+      composer global require wp-coding-standards/wpcs phpcompatibility/phpcompatibility-wp
```

Comment:
We should include a composer.json file in the plugin that has everything we need as a dependency to make sure that the same sniffs we run locally will be run in Travis.

Reply:
@joemcgill I will wait until https://github.com/humanmade/core-sitemaps/pull/15 has been merged before updating this as a `composer.json` file has been created as part of that PR